tests/code-check-roave-backward-compatibility/matrix.json

```diff
@@ -0,0 +1,10 @@
+{
+    "include": [
+        {
+            "name": "Backward Compatibility Check [@default, latest]",
+            "job": "{\"command\":\"roave-backward-compatibility-check check --from=\\\"1111222233334444aaaabbbbccccdddd\\\" --install-development-dependencies\",\"php\":\"@default\",\"extensions\":[],\"ini\":[],\"dependencies\":\"latest\",\"ignore_platform_reqs_8\":false,\"ignore_php_platform_requirement\":false,\"additional_composer_arguments\":[],\"before_script\":[]}",
```

Comment:
`--install-development-dependencies` should probably be dropped, but OK this way for a first iteration

Reply:
I just kept it as is, I can remove it if it makes no sense. 🤷🏼‍♂️ 
I have no clue what it does 😅

Reply:
The default behavior is `composer install --no-dev` while running the tool, so it will assume that any sources in `"autoload"` can be analyzed.

With this flag, it will remove the `--no-dev`, and run a full installation, with dev dependencies.

Thinking of it further, I believe we don't currently care that much: let's leave it there, for now :D